commcare_connect/utils/commcarehq_api.py

```diff
@@ -48,32 +50,39 @@ def refresh_access_token(user, force=False):
     return social_token
 
 
-def get_domains_for_user(user):
-    social_token = refresh_access_token(user)
+def get_domains_for_user(api_key):
     response = httpx.get(
-        f"{settings.COMMCARE_HQ_URL}/api/v0.5/user_domains/?limit=100",
-        headers={"Authorization": f"Bearer {social_token}"},
+        f"{api_key.hq_server.url}/api/v0.5/user_domains/?limit=100",
```

Comment:
does this api allow api key? if so, what does it do when the key is domain scoped?

Reply:
yeah, that API allows api keys auth. When a domain scoped API key is given it returns all the domains for the user.  🤦🏽